images/metering-ansible-operator/roles/meteringconfig/tasks/reconcile_presto.yml

```diff
@@ -49,6 +49,10 @@
       - template_file: templates/presto/presto-serviceaccount.yaml
         apis: [ {kind: serviceaccount} ]
         prune_label_value: presto-serviceaccount
+      - template_file: templates/presto/presto-prometheus-auth.yaml
+        apis: [ {kind: serviceaccount} ]
+        prune_label_value: presto-prometheus-auth
+        create: "{{ meteringconfig_create_presto_prometheus_auth }}"
```

Comment:
Hmm, I don't see this defined anywhere - if that's the case then just add this variable to the `defaults/main.yml` file somewhere above the `_ocp_enabled_overrides` dictionary, e.g.:
```yaml
meteringconfig_create_presto_prometheus_auth: "{{ _presto_spec.config.connectors.prometheus.enabled | default(false) }}"
```

Reply:
Done in 622ca1e, still passes `make verify`